docs/en/observability/synthetics-quickstart-fleet.asciidoc

```diff
@@ -0,0 +1,93 @@
+:synthetics-image: docker.elastic.co/beats/elastic-agent-complete:{version}
+
+[[synthetics-quickstart-fleet]]
+= Quickstart: Synthetic monitoring via {agent} and {fleet}
+
+++++
+<titleabbrev>Quickstart with Elastic Agent and Fleet</titleabbrev>
+++++
+
+[discrete]
+[[synthetics-quickstart-fleet-step-one]]
+== Step 1: Pull the latest {agent} complete docker image
+
+[source,sh,subs="attributes"]
+----
+docker pull {synthetics-image}
+----
+
+[discrete]
+[[synthetics-quickstart-fleet-step-two]]
+== Step 2: Set up Fleet Server
+
+Synthetic monitoring via {agent} and {fleet} requires {fleet-server} to be set up. To learn more, see https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html#set-up-fleet[{fleet}].
+
+[discrete]
+[[synthetics-quickstart-fleet-step-three]]
+== Step 3: Create an agent policy for Elastic Synthetics
+
+Use policies to manage settings across a group of {agent}s. A policy can contain any number of integrations for collecting observability data from the various services running on your host. {fleet} is automatically configured with a default agent policy, but it is useful to create a separate agent policy to use for synthetics tests. For more information on creating policies, see https://www.elastic.co/guide/en/fleet/current/agent-policy.html#create-a-policy[{agent} policy].
+
+[role="screenshot"]
+image::images/synthetics-agent-policy.png[Synthetics integration]
+
+[discrete]
+[[synthetics-quickstart-fleet-step-four]]
+== Step 4: Add an {agent} to {fleet}
+
+{agent} is a single, unified agent that you can deploy to hosts or containers to collect data and send it to the {stack}. Behind the scenes, {agent} runs {beats} or {elastic-endpoint} required for your configuration. 
+
+NOTE: To run synthetics test, you must use the *elastic-agent-complete* Docker container, which contains the dependencies to run synthetic monitors.
```

Comment:
Should test be plural here? If not, you should say, "To run a synthetics test..."

Reply:
Yep, should be plural
```suggestion
NOTE: To run synthetics tests, you must use the *elastic-agent-complete* Docker container, which contains the dependencies to run synthetic monitors.
```